Applications/Spire/Include/Spire/Ui/RealSpinBox.hpp

```diff
@@ -98,7 +98,8 @@ namespace Spire {
       QString display_string(Real value);
       QStyle::SubControl get_current_control(const QPoint& mouse_pos);
       boost::optional<Real> get_value(const QString& text) const;
-      bool is_valid(const QString& string);
+      bool is_valid_input(const QString& string) const;
```

Comment:
The technical term you're looking for is "well formed" which briefly speaking means that the input is syntactically valid but may or may not be semantically valid.

Hence rename ```is_valid_input``` to ```is_well_formed``` and ```is_valid_value``` to ```is_valid```.